src/components/DownloadFTP/Pfam/index.tsx

```diff
@@ -13,96 +12,57 @@ const PfamDownloads = () => (
-              Pfam-A models
-            </Link>
-          </td>
-          <td>
-            Pfam-A HMMs in an HMM library searchable with the hmmscan program.
-          </td>
-          <td className={css('xs-hide')}>Pfam-A.hmm.gz</td>
-          <td className={css('xs-hide')}>gzipped</td>
-          <td style={{ whiteSpace: 'nowrap' }}>
-            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.hmm.gz">
-              <span
-                className={css('icon', 'icon-common', 'font-l')}
-                data-icon="&#x3d;"
-              />
+              gzipped
             </Link>
-            <br />
           </td>
         </tr>
         <tr>
+          <td>Pfam-A HMM data</td>
           <td>
             <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.hmm.dat.gz">
-              Pfam-A HMM data
+              gzipped
             </Link>
           </td>
-          <td>Data associated with each HMM required for pfam_scan.pl</td>
-          <td className={css('xs-hide')}>Pfam-A.hmm.dat.gz</td>
-          <td className={css('xs-hide')}>gzipped</td>
-          <td style={{ whiteSpace: 'nowrap' }}>
-            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.hmm.dat.gz">
-              <span
-                className={css('icon', 'icon-common', 'font-l')}
-                data-icon="&#x3d;"
-              />
-            </Link>
-            <br />
-          </td>
         </tr>
         <tr>
+          <td>Pfam-A Seed alignment</td>
           <td>
             <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.seed.gz">
-              Pfam-A Seed alignment
+              gzipped
             </Link>
           </td>
+        </tr>
+        <tr>
+          <td>Pfam-A Full alignment</td>
           <td>
-            Annotation and seed alignments of all Pfam-A families in Pfam
-            format.
-          </td>
-          <td className={css('xs-hide')}>Pfam-A.seed.gz</td>
-          <td className={css('xs-hide')}>gzipped</td>
-          <td style={{ whiteSpace: 'nowrap' }}>
-            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.seed.gz">
-              <span
-                className={css('icon', 'icon-common', 'font-l')}
-                data-icon="&#x3d;"
-              />
+            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.full.gz">
+              gzipped
             </Link>
-            <br />
           </td>
         </tr>
         <tr>
+          <td>Checksums</td>
           <td>
-            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/Pfam-A.full.gz">
-              Pfam-A Full alignment
+            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/md5_checksums">
+              md5
             </Link>
           </td>
+        </tr>
```

Comment:
I wouldn't include the MD5 file here because it include checksums for files not listed on this page.